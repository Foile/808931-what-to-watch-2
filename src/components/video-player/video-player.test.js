import React from "react";
import renderer from "react-test-renderer";

import VideoPlayer from "./video-player";

it(`VideoPlayer correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<VideoPlayer
      src={``}
      poster={``}
      muted={true}
      width={280}
      height={175}
      playerState={{}}
    />);

  expect(tree).toMatchSnapshot();
});
