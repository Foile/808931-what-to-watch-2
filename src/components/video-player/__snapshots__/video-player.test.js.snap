// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer correctly renders after relaunch 1`] = `
<video
  height={175}
  muted={true}
  poster=""
  src=""
  width={280}
/>
`;
